Api/Domain/Validators/Exceptions/NotificationExceptions/EmailNofiticationException.cs

```diff
@@ -0,0 +1,7 @@
+namespace Api.Domain;
+
+public class EmailNofiticationException : Exception
+{
+    public EmailNofiticationException(string message) : base(message)
```

Comment:
I think the correct name is EmailNotificationException for dont make error in the future XD